packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -296,7 +296,7 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
       }
       // State type is private, hence using dynamic.
       dynamic getInspectorState() => inspectorKey.currentState;
-      String paragraphText(RenderParagraph paragraph) => paragraph.text.text;
+      String paragraphText(RenderParagraph paragraph) => (paragraph.text as TextSpan).text;
```

Comment:
how do we know `paragraph.text` is a `TextSpan`?

Reply:
Since this is an existing test, everything before this patch was a TextSpan. Should I add checking here for the future as well?

Reply:
Oh I didn't realise this was a test, I thought it was general purpose inspector code. If it's a test and we know it's the right type in that test, then this is fine.